stockflux-securities-master/src/reducers/securitiesTableReducer.js

```diff
@@ -0,0 +1,32 @@
+import { TableState } from "../enums";
+
+export const initialTableState = {
+  fetchStatus: TableState.LOADING,
+  hasErrors: false,
+  messages: []
+};
+
+export function securitiesTableReducer(state, action) {
+  switch (action.type) {
+    case "loading":
+      return { ...state, fetchStatus: TableState.LOADING, messages: [] };
```

Comment:
I think maybe you should reset `hasErrors` here to false as we are loading some new data in, so old error messages are no longer relevant.

Reply:
Yep I've added this in now